files/ru/mdn/tools/kumascript/troubleshooting/index.md

```diff
@@ -25,7 +25,7 @@ original_slug: MDN/Kuma/Troubleshooting_KumaScript_errors
 - Использование фигурных скобок без намерения вызвать макрос.
   - : Если вам нужно написать \\{ в документе, не вызывая макрос, вы можете экранировать его используя таким образом `\\{`
 - Использование специального символа в параметрах макроса.
-  - : Если вам нужно использовать " или \ внутри параметра макроса, они могут быть экранированы с помощью `\"` и `\\` соответственно 
+  - : Если вам нужно использовать " или \ внутри параметра макроса, они могут быть экранированы с помощью `\"` и `\\` соответственно
```

Comment:
It seems that this change is unrelated/the wrong locale? I don't really mind keeping it, but the review process might be a little bit slower.

Reply:
right, that was unintentional, I'll rebase it out